lib/rex/post/meterpreter/ui/console/command_dispatcher/core.rb

```diff
@@ -1208,7 +1208,7 @@ def cmd_migrate(*args)
 
     begin
       server = client.sys.process.open
-    rescue TimeoutError => e
+    rescue Rex::TimeoutError, ::Timeout::Error => e
```

Comment:
Presumably this was intended to catch the exceptions raised by `lib/rex/post/meterpreter/packet_dispatcher.rb` within the `send_request` method - fixed above. I've decided to catch both Rex::TimeoutError and Timeout::Error here for simplicity.